.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "stage": 1,
+  "loose": true,
+  "optional"  : [
+    "utility.inlineEnvironmentVariables"
+  ]
+}
```

Comment:
I'm curious about other options for this; .rc files feel magical to me. Is there a way to specify these options when running babel itself, or is this just the cleanest way?